### DISCUSSION TITLE
Merging housings

### DISCUSSION THREAD

**Discussion by vmichel** [streetcomplete/StreetComplete#3501] [Feedback]
Hi, 

Is it possible to merge  multiple parts of building that are considered as separated housings (e.g. balcony, dependancies...) but should considered within the main building ?
There is often 2 or 3 entries to fill for one building and it does not make sense. An option to merge them to the main part in the App will be really nice. Thanks

**Comment by matkoniecz** [comment]
For SC mapper: leave note in such case, preferably with a photo.

---------------------------
Is it possible to handle it in SC?

Not really, as it requires inspection of all visible tags and different action may be needed depending on various factors.

Sometimes it requires changing `building=yes` to `building:part=yes` https://wiki.openstreetmap.org/wiki/Key:building:part

Sometimes elements needs to be deleted.

Sometimes `building=yes` needs to be removed.

Sometimes new building geometry needs to be created.

Sometimes more than one of things above needs to be done.

In addition addresses may need to be extracted from `building=yes` areas into address nodes.

And other possibilities that I forgot right now.

**Reply by vmichel** [reply]
Understood, thanks for your quick reply, indeed it seems more complicated to handle as I thought. Thanks for your work anyway !
